packages/design-picker/src/components/style.scss

```diff
@@ -406,14 +406,46 @@
 .design-picker.design-picker__unified {
 	display: block;
 	padding-bottom: 200px;
+
 	.responsive-toolbar-group__dropdown .responsive-toolbar-group__grouped-list {
 		justify-content: flex-start;
 	}
+
 	.design-picker__grid {
 		@supports ( display: grid ) {
+			row-gap: 0;
+
 			@include break-medium {
 				grid-template-columns: 1fr 1fr 1fr;
 				column-gap: 24px;
+
+				.design-button-container:nth-child(-n+3) {
+					margin-top: 0;
+				}
+			}
+
+			.design-button-container {
+				margin-top: 32px;
```

Comment:
I think we have to remove this margin from the `:first-child` for all breakpoints.
|Before|After|
|------|-----|
|<img width="393" alt="Screen Shot 2565-10-28 at 16 16 33" src="https://user-images.githubusercontent.com/1881481/198552216-dc1aa49b-c009-46e1-92cf-ded08a83fdce.png">|<img width="392" alt="Screen Shot 2565-10-28 at 16 16 26" src="https://user-images.githubusercontent.com/1881481/198552230-36b588dd-cb9f-4814-9cce-2a2a20bb8de4.png">|

Reply:
Good catch! Updated.